packages/bun-types/bun.d.ts

```diff
@@ -3753,6 +3753,43 @@ declare module "bun" {
    */
   function sleepSync(ms: number): void;
 
+  /**
+   * Parse a time string and return milliseconds, or format milliseconds as a string.
+   *
+   * Drop-in replacement for the `ms` npm package with compile-time inlining support.
+   *
+   * @example
+   * ```ts
+   * Bun.ms("2d")      // 172800000
+   * Bun.ms("1.5h")    // 5400000
+   * Bun.ms("1m")      // 60000
+   * Bun.ms("5s")      // 5000
+   * Bun.ms("100ms")   // 100
+   * Bun.ms("1mo")     // 2629800000
+   * Bun.ms("1y")      // 31557600000
+   * Bun.ms("100")     // 100
+   * Bun.ms(60000)     // "1m"
+   * Bun.ms(60000, { long: true })  // "1 minute"
+   * Bun.ms("invalid") // NaN
+   * ```
+   *
+   * Supports these units:
+   * - `ms`, `millisecond`, `milliseconds`
+   * - `s`, `sec`, `second`, `seconds`
+   * - `m`, `min`, `minute`, `minutes`
+   * - `h`, `hr`, `hour`, `hours`
+   * - `d`, `day`, `days`
+   * - `w`, `week`, `weeks`
+   * - `mo`, `month`, `months`
+   * - `y`, `yr`, `year`, `years`
+   *
+   * @param value - Time string to parse or milliseconds to format
+   * @param options - Formatting options (when value is a number)
+   * @returns Milliseconds (for string) or formatted string (for number). Returns NaN for invalid strings.
+   */
+  function ms(value: string): number;
+  function ms(value: number, options?: { long?: boolean }): string;
```

Comment:
Why not separate out the two overloads' JSDoc?

The authority himself has spoken: 
<img width="435" height="202" alt="image" src="https://github.com/user-attachments/assets/87d4e7fa-c0bd-4199-839e-92714c86c899" />